ui/k-toolbar/main.kytos

```diff
@@ -168,6 +180,39 @@ module.exports = {
       });
 
     },
+    async fetch_dpids() {
+      /*
+      fetches a list of DPIDs (Datapath IDs) and interface information from an API endpoint, 
+      then maps the DPIDs and interface names to a formatted string, 
+      and finally sets the resulting array of strings as the dpids property of the component it belongs to. 
+      If an error occurs during the fetch, it logs the error to the console
+      */
+  try {
+    const response = await fetch('/api/kytos/topology/v3/interfaces');
+    const data = await response.json();
+    const dpids = Object.entries(data.interfaces).map(([key, value]) => {
+      let item = key;
+      if (value.name) {
+        item = `${value.name} - ${item}`;
```

Comment:
Could you also cover this part of the original requirement to preferably set it with `metadata.port_name` if it's defined otherwise using the same format that you're using here, let me quote the requirement:

> and the name of the displayed interfaces should be preferably `${interface_obj.metadata.port_name} - ${id`}, if the `metadata.port_name` is set and not empty, otherwise it should be `${interface_obj.name} - ${id`}: